backend/src/main/java/ca/bc/gov/hlth/hnweb/controller/GroupMemberController.java

```diff
@@ -112,7 +114,11 @@ public ResponseEntity<UpdateNumberAndDeptResponse> updateNumberAndDept(@Valid @R
 		try {	
 			// Handle the department number
 			UpdateNumberAndDeptResponse deptNumberResponse = new UpdateNumberAndDeptResponse();
-			if (StringUtils.isNotBlank(updateNumberAndDeptRequest.getDepartmentNumber())) {
+			String deptNumber = updateNumberAndDeptRequest.getDepartmentNumber();
+			if (StringUtils.isNotBlank(deptNumber)) {
+				if(deptNumber.length() >= 6 && deptNumber.substring(0,6).contentEquals(SIX_DASHES)) {
```

Comment:
Missing space after opening parenthesis. Also, I think this needs to be an exact match on the dashes. The current code would accept '------ABC' from an API call (even though the UI wouldn't allow this).